pom.xml

```diff
@@ -1178,9 +1178,9 @@
             </dependency>
 
             <dependency>
-                <groupId>mysql</groupId>
-                <artifactId>mysql-connector-java</artifactId>
-                <version>5.1.48</version>
+                <groupId>com.mysql</groupId>
+                <artifactId>mysql-connector-j</artifactId>
+                <version>8.1.0</version>
```

Comment:
I see latest version in `8.2.0` [here](https://mvnrepository.com/artifact/com.mysql/mysql-connector-j) 
Any reason, we are not moving to that?

Reply:
No specific reason i can try with 8.2.0 as well.